config/config_loader.go

```diff
@@ -59,6 +59,8 @@ var (
 	confBaseFile                    string
 	uniformVirtualServiceConfigPath string
 	uniformDestRuleConfigPath       string
+
+	loaderHooks map[string][]LoaderHook
```

Comment:
Why not initializes `loaderHooks` here? If not initializes, you should handle the case that `loaderHooks` is empty in  `AddLoaderHooks` and `RemoveLoaderHooks` method.